blog/blocks/interlinks/interlinks.js

```diff
@@ -61,10 +61,10 @@ export default async function interlink() {
       .forEach((p) => {
         // set paragraph link limit: 1 every 40 words
         const paraLinks = p.querySelectorAll('a').length;
-        const paraWords = p.textContent.split(/\s/).length;
+        const paraWords = p.textContent.trim().split(/\s/).length;
         const maxParaLinks = Math.floor(paraWords / 40) - paraLinks;
         if (maxParaLinks > 0) {
-          Array.from(p.childNodes)
+          Array.from(p.children)
```

Comment:
@jckautzmann this is actually a case where you _want_ to iterate over the text nodes (see futher below):
```suggestion
          Array.from(p.childNodes)
```

Reply:
ah right, reverted this change with https://github.com/adobe/business-website/pull/235/commits/85472aead4b70c63ff369c1ab913c67d51565f20